manticore/platforms/linux.py

```diff
@@ -826,7 +832,10 @@ def load(self, filename):
                 continue
             interpreter_filename = elf_segment.data()[:-1]
             logger.info('Interpreter filename: %s', interpreter_filename)
-            interpreter = ELFFile(file(interpreter_filename))
+            try:
+                interpreter = ELFFile(file(interpreter_filename))
+            except:
+                pass
```

Comment:
Why ignore the exception?

Reply:
My mistake : I will just use the code from python3 branch